content/docs/reference-events.md

```diff
@@ -149,53 +149,53 @@ boolean shiftKey
 number which
 ```
 
-The `key` property can take any of the values documented in the [DOM Level 3 Events spec](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
+A propriedade `key` pode conter quaisquer valores documentados na [especificação de eventos do DOM Level 3](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
```

Comment:
I believe that "a propriedade `key` pode receber..." may be a better translation to "_the `key` property can take..._", since you have to pass the property value to it. What do you think?

```suggestion
A propriedade `key` pode receber quaisquer valores documentados na [especificação de eventos do DOM Level 3](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
```

Reply:
Exatamente. +1 para "take" como "receber", nesse contexto.